superset/assets/src/visualizations/Table/buildQuery.ts

```diff
@@ -0,0 +1,47 @@
+import buildQueryContext from 'src/query';
+import { Metrics, RawMetric } from 'src/query/Metric';
+import FormData from './FormData';
+
+export default function buildQuery(formData: FormData) {
+  // Set the single QueryObject's groupby field with series in formData
+  return buildQueryContext(formData, (baseQueryObject) => {
```

Comment:
Haven't reviewed the new `buildQuery` for table view. Can take another pass after we address the type related comments above.